source/agora/node/Registry.d

```diff
@@ -540,6 +540,70 @@ private struct TypedPayload
     /// UTXO
     public Hash utxo;
 
+    /***************************************************************************
+
+        Construct an instance of an `TypedPayload` from REST payload
+
+        Params:
+            type = Resource record type
+            payload = Payload data of API
+            utxo = UTXO of the validator (optional)
+
+    ***************************************************************************/
+
+    public this (TYPE type, RegistryPayload payload, Hash utxo = Hash.init) @safe
```

Comment:
Missing other attributes.
Another approach would be to rename the other ctor into `make` (a `static` function) so that you keep the default ctor. I wish we had something like:
```D
public this () = default;
```

as C++ does :(